pkg/utils/utils.go

```diff
@@ -52,6 +58,45 @@ func ColoredString(str string, colorAttribute color.Attribute) string {
 	return ColoredStringDirect(str, colour)
 }
 
+// ColoredYamlString takes an YAML formatted string and returns a colored string
+// with colors hardcoded as:
+// keys: cyan
+// Booleans: magenta
+// Numbers: yellow
+// Strings: green
+func ColoredYamlString(str string) string {
```

Comment:
Can you add a unit test for this? (Accidentally just left this as a general comment cos I'm reviewing from phone :) )

Reply:
@jesseduffield sure, done

Reply:
And sadly, i don't get about scrapping "the JSON-related code" :c

Reply:
What I mean is, let's remove `DetailsFormat` and always use yaml formatting.